test/containers/trends/SMBGRangeAvgAnimationContainer.test.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import React from 'react';
+
+import { mount } from 'enzyme';
+
+import * as scales from '../../helpers/scales';
+const {
+  trendsXScale: xScale,
+  trendsYScale: yScale,
+} = scales.trends;
+
+import SMBGRangeAvgAnimationContainer
+  from '../../../src/containers/trends/SMBGRangeAvgAnimationContainer';
+
+describe('SMBGRangeAvgAnimationContainer', () => {
+  let wrapper;
+
+  // six-hour bins for testing
+  const binSize = 1000 * 60 * 60 * 6;
```

Comment:
just noticed else where you where doing `THREE_HRS * 6` should we just do the same here as it is a multiple of 3?